src/main/java/seedu/rc4hdb/model/venues/Venue.java

```diff
@@ -111,9 +112,9 @@ public boolean isSameVenue(VenueName otherVenueName) {
 
     //==================== Start of getters =================================
 
-    public List<Booking> getBookings() {
+    public List<Booking> getReadOnlyBookings() {
         clearExpiredBookings();
-        return bookings;
+        return Collections.unmodifiableList(bookings);
```

Comment:
Good job spotting this. Does the resulting list get garbage collected like the one returned from `FXCollections::unmodifiableList`, if no reference is held to it?